optimade/validator/validator.py

```diff
@@ -104,11 +104,14 @@ def __init__(self, base_url: str, max_retries=5):
 
         Parameters:
             base_url (str): the base URL of the optimade implementation, including
-            request protocol (e.g. `'http://'`) and API version number if necessary.
-            Examples:
-                - `'http://example.org/optimade/v1'`,
+                request protocol (e.g. `'http://'`) and API version number if necessary.
+
+                Examples:
+
+                - `'http://example.org/optimade/v1'`
                 - `'www.crystallography.net/cod-test/optimade/v0.10.0/'`
-            Note: A maximum of one slash ("/") is allowed as the last character.
+
+                Note: A maximum of one slash ("/") is allowed as the last character.
```

Comment:
Commenting here as GH won't let me comment elsewhere, the `Drawbacks:` part of the [`ImplementationValidator` docstring](https://github.com/Materials-Consortia/optimade-python-tools/blob/5c04e51011f64ceb691ca3c773fed58f7942cad6/optimade/validator/validator.py#L293) gets rendered as an admonition by mkdocs `!!! drawbacks`, but it doesn't exist for this theme, so we could change it to `Caution:` or `Warning:` which I think will work?

Reply:
It doesn't seem to work locally for me, unfortunately...

Reply:
So this is an example where the first line needs to be taken down (referring to your comment above concerning the syntax).
It seems for class doc strings it doesn't read it properly unless nothing follows the first `"""`.

Reply:
![image](https://user-images.githubusercontent.com/43357585/88952654-04772300-d298-11ea-9e1e-d722d80b1e50.png)

Reply:
![image](https://user-images.githubusercontent.com/43357585/88952757-2670a580-d298-11ea-8a16-9c9749c41e30.png)

Reply:
There's some extra magic happening here somewhere, as the PR stands, just having the line

```
Drawbacks:
    blah, blah
```

leads to mkdocs generating

```
!!! drawbacks
    blah, blah, blah
```

It would be nice if we didn't have to litter out docstrings with `!!!`, if you know the way of handling this? It might just be the newline as you suggest

Reply:
This generates a "warning" in the docs for me, I'm sure "Caution:" would also work. Just had to add the newline, as you suggested:

![image](https://user-images.githubusercontent.com/7916000/88953249-89f6d500-d290-11ea-8a92-39532f7210f3.png)

Reply:
(doesn't need to be a list either)

Reply:
See my suggestion in https://github.com/Materials-Consortia/optimade-python-tools/pull/430#discussion_r463149884

Reply:
It actually looks like you can write any word in there, "Drawbacks:" now works fine too, it just makes it note-style rather than  warning-style.

Reply:
Yeah, so if the admonition is not recognized it simply creates a note, i.e.:
```markdown
Whatever:
    Some text.
```
becomes:
```markdown
!!! note "Whatever"
    Some text.
```

Reply:
> This generates a "warning" in the docs for me, I'm sure "Caution:" would also work. Just had to add the newline, as you suggested:

Yup, so for class doc strings this is apparently an issue. It may however be solved in the newest `pytkdocs` version, since it parses the doc strings in a different way. But I'm not completely sure.

Reply:
So again, if you want the box to say "Drawbacks" instead of "Caution" with a caution admonition, you need to write it out explicitly like I did.